package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "el-gallo-giro",
+  "version": "1.0.0",
+  "description": "Sebas first business website",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "npx stylelint css/**.css --fix"
```

Comment:
So this is the magic line right here. So here's what you'll do. 
1. [First you need to have node/npm install if you don't already](https://www.newline.co/@Adele/how-to-install-nodejs-and-npm-on-macos--22782681)
2. Once installed run the command `npm run lint` in your terminal and you should automatically see all the code change automatically for you. You'll want to commit those changes.
3. You'll also want to [install this extension in vscode](https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint) so you can see your lint errors